assignment5.txt

```diff
@@ -0,0 +1,18 @@
+Jinoo Hong 
+
+1.  What do each of the following mean?  Include a description of the syntax or an example.
+      - Passing by value:
+		Passing by value means that the function will be using a copy of the argument inputted and so this will not affect the value of the original variable. (most local variables) "auto int x = 1;"
+
+
+      - Passing by reference:
+		Passing by reference means that the function will be given the actual memory address of the variable passed and so will directly be changing the original variable. (global variables and arrays in c) "static int x = 1;"
```

Comment:
Right - note that every parameter passed in C is by value! See http://stackoverflow.com/questions/12953496/what-happens-during-pass-by-reference-in-c